addons/account/models/account_move.py

```diff
@@ -2053,6 +2056,26 @@ def _compute_next_payment_date(self):
         for move in self:
             move.next_payment_date = min([line.payment_date for line in move.line_ids.filtered(lambda l: l.payment_date and not l.reconciled)], default=False)
 
+    @api.depends('fiscal_position_id')
+    def _compute_unaligned_taxes_fp_warning(self):
+        """
+        Method to compute whether the applied taxes align with the fiscal position of the move.
+        """
+
+        for move in self:
+            if move.state == 'draft' and move.is_invoice(include_receipts=True) and move.fiscal_position_id:
+                # Avoid iterating over all records by stopping at the first mismatch.
+                has_unaligned_tax = any(
+                    not all(tax in line.tax_ids for tax in line._get_computed_taxes())
```

Comment:
```suggestion
                    any(tax not in line.tax_ids for tax in line._get_computed_taxes())
```
use any rather using all
Also it would not work expected in case `_get_computed_taxes` returned a `Falsy` value it would be already a False and using a not on the outer hand would had make it True